wordninja.py

```diff
@@ -2,40 +2,50 @@
 from math import log
 
 
-
-__version__ = '0.1.4'
+__version__ = '0.1.5'
 
 
 # I did not author this code, only tweaked it from:
 # http://stackoverflow.com/a/11642687/2449774
 # Thanks Generic Human!
 
 
+# Modifications by Scott Randal (Genesys)
+#
+# 1. Preserve original character case after splitting
+# 2. Avoid splitting every post-digit character in a mixed string (e.g. 'win32intel')
+# 3. Avoid splitting digit sequences
+# 4. Handle input containing apostrophes (for possessives and contractions)
+#
+# Dictionary changes:
```

Comment:
It's not really a dictionary -- just a wordlist. I can change this to 'changes to wordninja_words.txt.gz